ddl/ddl_api.go

```diff
@@ -955,6 +955,11 @@ func buildTableInfo(ctx sessionctx.Context, d *ddl, tableName model.CIStr, cols
 				}
 			}
 		}
+		if constr.Tp == ast.ConstraintFulltext {
```

Comment:
@shenli Should we ignore fulltext index?

Reply:
@shenli @winkyao the current behavior is to convert a `fulltext` index to a regular index. In the referenced issue, this prevents some mysqldump imports, because it leads to trying to index blob/text directly (unsupported).  The new behavior skips adding the index, and provides a warning (warnings are better for DDL than errors).